geo_autoRIFT/geogrid/src/geogridOptical.cpp

```diff
@@ -1040,8 +1077,8 @@ void geoGridOptical::geogridOptical()
                         }
                         else
                         {
-                            raster11[jj] = std::round(dot_C(vel,xunit)*dt/XSize/365.0/24.0/3600.0*1);
-                            raster22[jj] = std::round(dot_C(vel,yunit)*dt/YSize/365.0/24.0/3600.0*1);
+                            raster11[jj] = std::round(dot_C(vel,xunit)*dt/norm_C(xdiff)/365.0/24.0/3600.0*1);
+                            raster22[jj] = std::round(dot_C(vel,yunit)*dt/norm_C(ydiff)/365.0/24.0/3600.0*1);
```

Comment:
Are raster11 and raster22 used for optical processing? Why did you need to change from grid size to norm?

Reply:
@alex-s-gardner raster11 and raster22 are downstream search pixel distances, so not related to the conversion matrix. Grid size (XSize and YSize) was defined in image CS (input projection for optical), while norm(xdiff) and norm(ydiff) are transformed pixel sizes in map projection CS (output projection for optical). Again this is related to calculating the downstream search pixel distances: when vel and xunit/yunit are all defined in output projection, the pixel sizes should be also in that CS rather than the input one.